### PR TITLE
PEER-252 Dockerize question service

### PR DESCRIPTION
**Pull request by anunayajoshi**

### CODE REVIEW

File: backend/question/.env.compose
Comment (SeeuSim) [inline backend/question/.env.compose:1]: unquote all strings

File: backend/question/docker-compose.yml
Comment (SeeuSim) [inline backend/question/docker-compose.yml:1]: We will no longer be using this, make changes in main config instead

**Review by SeeuSim — CHANGES_REQUESTED** [review]
Verify that both:
- standalone dockerfile build and run with local db cluster
- integrated docker compose build 
will run.

Add in configs in Docker-Compose main config, as well as screenshots of successful curl requests for both setups

File: backend/question/.env.compose
Comment (SeeuSim) [reply backend/question/.env.compose:1]: Ignore this review

**Comment by anunayajoshi** [comment]
For combined:
image
image


For standalone using question-express-local:

File: backend/question/.env.local [hunk 1]
```diff
@@ -3,7 +3,7 @@ PEERPREP_UI_HOST="http://localhost:5173"
 
 EXPRESS_PORT=9002
 EXPRESS_DB_HOST="localhost"
-EXPRESS_DB_PORT=5433
+EXPRESS_DB_PORT=5432
```
Comment (SeeuSim) [inline backend/question/.env.local]: Revert this change. 5432 is a reserved port.

**Review by SeeuSim — CHANGES_REQUESTED** [review]

**Review by SeeuSim — APPROVED** [review]
lgtm
